README.md

```diff
@@ -15,5 +15,23 @@ docker run \
   ibmcom/mq
 ```
 
+Og sette opp en database lokalt:
+```
+docker run --name familie-oppdrag -e POSTGRES_PASSWORD=test -d -p 5432:5432 postgres
+docker ps (finn container id)
+docker exec -it <container_id> bash
+psql -U postgres
+CREATE DATABASE "familie-oppdrag";
+```
+
+For å kjøre med denne lokalt må følgende miljøvariabler settes i `application-local.yml`:
+```
+spring.datasource.url=jdbc:postgresql://0.0.0.0:5432/familie-oppdrag
+spring.datasource.username=postgres
+spring.datasource.password=test
```

Comment:
Klarer appen å kjøre lokalt uten at man setter opp postgres? Via en in-memory H2-base feks? Det hadde kanskje vært nice å slippe å sette opp postgres lokalt hvis man jobber med noe helt urelatert til basen. Men hvis postgres alltid må være oppe lokalt, så kan de miljøvariablene kanskje ligge permanent i application-dev?